contributing/code/bc.rst

```diff
@@ -277,6 +277,12 @@ Change value of a constant                          Yes [1]_ [5]_
        Additionally, if a constant will likely be used in objects that are
        serialized, the value of a constant should not be changed.
 
+.. [6] Allowed using the ``@final`` annotation.
+
+.. [7] Allowed if the class is final.
```

Comment:
In fact I don't see how we could clearly explain that the `@final` annotation means that the class/method is final but only if it is present since the previous major version, I think it is clear enough as is.